www/js/toolbar.js

```diff
@@ -78,36 +80,39 @@ const switchFactory = (id, label, inputId, clickEvent, defaultValue = true) =>
     ]),
   ]);
 
+const syncToggle = async (forceConnect = false) => {
+  if (isConntected && !forceConnect) {
+    isConntected = false;
+    onEnd(code);
+    code = '...';
+    global.controller.sendText('');
+    document.querySelector('.sync-code-num').innerText = '...';
+    document.querySelector('#tool-sync-button').setAttribute('title', '...');
+  } else {
+    isConntected = true;
+    await remoteSyncInit();
+  }
+  document.querySelector('.present-btn').innerText = isConntected
+    ? 'Stop Session'
+    : 'Start Session';
+};
+
 const syncContent = h('div.sync-content', [
   h('div.left-sync-content', [
-    h('div.left-sync-icon'),
-    h('button.button.left-sync-button', 'Watch Video'),
-  ]),
-  h('div.right-sync-content', [
-    h('div.sync-code-label', 'Your unique sync code is'),
-    h('div.sync-code-num', code),
     h(
       'div.sync-code-desc',
       'Enter this code on sttm.co/sync to follow along SikhiToTheMax on any web browser (including mobile). Press the "Present" button to display the pairing code for the sangat.',
     ),
+  ]),
+  h('div.right-sync-content', [
```

Comment:
can we make it so the popout closes clicking outside of the content? having the go the close button is far mouse travel.

Reply:
+1